remaining packer content (#13285)
hashicorp/packer: website/content/docs/commands/hcl2_upgrade.mdx

---
description: |

>   The `packer hcl2_upgrade` Packer command transpiles a JSON
>   configuration template to HCL2 so that you can transition to HCL templates.

Reviewer: ```suggestion
  The `packer hcl2_upgrade` Packer command transpiles a JSON
  configuration template into HCL2, helping you migrate to using HCL templates.
```
